config/jobs/ppc64le-cloud/ibmcloud-postsubmit.yaml

```diff
@@ -0,0 +1,56 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-ibmcloud-job
+      decorate: true
+      run_if_changed: 'images/ibm-cloud'
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        serviceAccountName: build-img
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.1
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+                export PATH=$GOPATH/bin:$PATH
+                export GO111MODULE=on
+                curl -L "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/`go env GOARCH`/kubectl" -o /usr/local/bin/kubectl
+                chmod +x /usr/local/bin/kubectl
+                pushd $GOPATH/src/github.com/vmware-tanzu/buildkit-cli-for-kubectl
+                make build install
+                popd
+                ca=$(base64 -w 0 /run/secrets/kubernetes.io/serviceaccount/ca.crt)
+                token=$(cat /run/secrets/kubernetes.io/serviceaccount/token)
+                namespace=$(cat /run/secrets/kubernetes.io/serviceaccount/namespace)
+                cat >kubeconfig <<EOL
+                apiVersion: v1
+                kind: Config
+                clusters:
+                - name: default-cluster
+                  cluster:
+                    certificate-authority-data: ${ca}
+                    server: https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT_HTTPS}
+                contexts:
+                - name: default-context
+                  context:
+                    cluster: default-cluster
+                    namespace: ${namespace}
+                    user: default-user
+                current-context: default-context
+                users:
+                - name: default-user
+                  user:
+                    token: ${token}
+                EOL
+                export KUBECONFIG=$(pwd)/kubeconfig
+                pushd images/ibm-cloud
+                kubectl build --push --registry-secret quay-powercloud-regcred --namespace image-builder --build-arg ARCH=amd64 -t quay.io/powercloud/ibm-cloud:amd64-latest -f Dockerfile ./
```

Comment:
- why do we need `--build-arg ARCH=amd64`?
- we don't need `amd64` in the image tag